README.md

```diff
@@ -1634,6 +1634,7 @@ API | Description | Auth | HTTPS | CORS |
 | [ClickMeter](https://support.clickmeter.com/hc/en-us/categories/201474986) | Monitor, compare and optimize your marketing links | `apiKey` | Yes | Unknown |
 | [Clico](https://cli.com/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config) | URL shortener service | `apiKey` | Yes | Unknown |
 | [Cutt.ly](https://cutt.ly/api-documentation/cuttly-links-api) | URL shortener service | `apiKey` | Yes | Unknown |
+| [Drivet URL Shortener](https://wiki.drivet.xyz/e/en/url-shortener/add-links) | Shorten a long URL easily and fast | No | Yes | Unknown |
```

Comment:
@GhostSlayer Current link still not working. Change URL from current one to https://wiki.drivet.xyz/en/url-shortener/add-links